tests/plugins/widget/integration/table/manual/widgetintable.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 1027, 4.8.0, widget
+@bender-ui: collapsed
+@bender-ckeditor-plugins: undo,clipboard,basicstyles,toolbar,wysiwygarea,widget,table,resize
+
+----
+
+1. Click into widgets in table
+1. Try to select some text **in table**
+1. Perform some operation on selected text ( change, append new text, apply some style, etc. )
+1. Check behaviour inside inner and outer table depend of testing widget
+1. Try to add new widget in table ( buttons without icon in toolbr )
```

Comment:
This looks a little out of place here. We can treat it as a last step of the scenario, but then it misses **expected** result. You can:

* Extract it as a second test scenario (well, pretty short one).
* Rearrange scenario in a way that the user/tester should add a widget on the beginning and then proceed with the rest of the steps (so added widget will also be a part of further testing).